src/main/java/org/sonar/plugins/stash/issue/MarkdownPrinter.java

```diff
@@ -120,12 +116,49 @@ private String formatTableList(String caption, List<String> items) {
     sb.append("|");
     sb.append(NEW_LINE);
     for (String item : items) {
-      sb.append("| ").append(item).append(" |").append(NEW_LINE);
+        sb.append("| ").append(item).append(" |").append(NEW_LINE);
+        if (this.includeFilesInOverview && issueToFilesMap.containsKey(item)) {
+            String files = String.join(", ", issueToFilesMap.get(item));
+            sb.append("| *Files: ").append(files).append("* |").append(NEW_LINE);
+        }
     }
 
     return sb.toString();
   }
 
+  private HashMap<String, ArrayList<String>> getMappedIssuesToFiles(Collection<PostJobIssue> allIssues) {
+    HashMap<String, ArrayList<String>> issueToFilesMap = new HashMap<>();
+    if (!this.includeFilesInOverview) {
+      return issueToFilesMap;
+    }
+
+    for (PostJobIssue issue : allIssues) {
+      String title = this.printIssueMarkdown(issue);
+      if (!issueToFilesMap.containsKey(title)) {
+        issueToFilesMap.put(title, new ArrayList<>());
+      }
+
+      if (issueToFilesMap.get(title).size() >= MAX_FILES_PER_ISSUE) {
+        continue;

```

Comment:
I think it would be more useful to print the first `MAX_FILES_PER_ISSUE` and then add an ellipsis.
(Except if the setting is `0`)

What do you think?
(In any case there should be some kind of indication that something has been left out)

Reply:
Yes, I think that makes sense.